liquid_node/jobs/bbb.py

```diff
@@ -0,0 +1,23 @@
+from liquid_node import jobs
+
+
+class BBB(jobs.Job):
+    name = 'bbb'
+    template = jobs.TEMPLATES / f'{name}.nomad'
+    app = 'bbb'
+    stage = 1
+    vault_secret_keys = [
+        "liquid/bbb/bbb.postgres",
+        "liquid/bbb/bbb.secret_key_base",
+    ]
+    core_oauth_apps = [
+        {
+            'name': 'bbb',
+            'subdomain': 'bbb',
+            'vault_path': 'liquid/bbb/auth.oauth2',
+            'callback': '/auth/openid_connect/callback',
+            'algo': 'HS256',
```

Comment:
you added these `algo` and `noskip` flags here, but i don't see you actually using them with your[ `--algo` and `--noskip` ](https://github.com/liquidinvestigations/core/pull/214/files) arguments here -- https://github.com/liquidinvestigations/node/blob/bbb/liquid_node/commands.py#L212


This probably means this code won't run on a fresh installation, since it won't give the arguments to the initial oauth2 app creation. After fixing  all the suggestions in these PRs, please wipe all the data from your server (or spin up a new VM) and do the whole setup from scratch following your own docs - that should catch some more similar problems.

Reply:
indeed I missed pushing some code from liquid_node/commands.py, I'll add that right away.